Streetcode/Streetcode.BLL/Interfaces/Audio/IAudioService.cs

```diff
@@ -0,0 +1,6 @@
+namespace Streetcode.BLL.Interfaces.Audio;
+
+public interface IAudioService
+{
+    public Task CleanUnusedAudios();
```

Comment:
The methods which return Task should have Async in their names
```suggestion
    public Task CleanUnusedAudiosAsync();
```